### PR TITLE
Material Design + NPM dependency

### PR DESCRIPTION
**Pull request by GuillaumeUnice**
Hi,

This PR add Angular Material Design Theme, and NPM to manage js dependency.
You just have to execute the following command:
```
cd webroot/scripts/
npm i
```

Result screen:
http://hpics.li/726bd68
http://hpics.li/1e0ce5b
http://hpics.li/4366682
http://hpics.li/6144517
http://hpics.li/a3fca40

### CODE REVIEW

File: webroot/scripts/NewsFeedCtrl.js [hunk 1]
```diff
@@ -16,21 +13,24 @@ vertxFeeds.controller('NewsFeedCtrl', ['$scope', '$http', function($scope, $http
 				var subscription = $scope.subscriptions[i];
 				console.log("register reader for : " + subscription.hash + " on the event bus");
 				eb.registerHandler(subscription.hash, function(entries){
+					console.log(entries);
```
Comment (aesteve) [inline webroot/scripts/NewsFeedCtrl.js]: could you please remove this ?

**Review by aesteve — COMMENTED** [review]
Thanks

**Comment by aesteve** [comment]
Thanks a lot for the Pull Request.


The only thing I'm really thinking about is that the example was nice because it was working out of the box, without any external requirement. Now, to get started, you have to have node installed, type `npm i`, wait, then start the whole thing.

This makes it a little bit more complex to understand what's going on for a starter.

Obviously anyone who has already worked with any serious front-end application knows this by heart, but for a minimalist example, I'm not convinced this is accurate.

What do you think ?

**Comment by GuillaumeUnice** [comment]
Hi,

It's not a problem all dependencies or fixed we just have to update the Readme, but, if you wouldn't use it, we could remove this operation.

Two solutions:
-Commit node_modules directory thus everyone have already dependency installed and as the projet is simple there is no lot of dependency to download.
-Remove NPM and search all library in a CDN.

What do you prefer?

**Comment by aesteve** [comment]
Hi and thanks for your answer. I'd prefer the CDN way of possible, that'd
be awesome.

It also puts the emphasis on the fact that it's just an example and should
not be used in production.

Thanks again.

**Comment by GuillaumeUnice** [comment]
Hi,
Ok I remove NPM and clean code.
All it's right :D

**Pull request merged**